Makefile

```diff
@@ -1,6 +1,6 @@
 
-THRIFT_VER=0.9.2
-THRIFT_IMG=thrift:$(THRIFT_VER)
+THRIFT_VER=0.13
+THRIFT_IMG=jaegertracing/thrift:$(THRIFT_VER)
```

Comment:
These two should be made configurable to allow submodules modify the build.

```
THRIFT_VER ?= 
```

Reply:
Currently only the directories are configurable. I currently reworked a lot of the `Makefile` in https://github.com/jaegertracing/jaeger-idl/pull/63 but this still doesn't make it configurable.

We might want to have a separate PR for making all docker versions configurable. I tried to keep the change in this PR minimal.

Reply:
>  I tried to keep the change in this PR minimal.

The changes I am asking for are for the lines that are changed in this PR.

I am not asking to change other docker versions just this one. The issue is that the `jaegertracing/thrift` contains only a single version.

Reply:
I'm fine with it. Adjusted it accordingly :) Might still be a good idea to discuss having all the others also configurable in a separate PR once this is in.